sonic_platform_base/sonic_xcvr/api/public/c_cmis.py

```diff
@@ -388,362 +388,53 @@ def get_transceiver_bulk_status(self):
-        txpower                      = FLOAT                            ; tx power in mW
-        rxpower                      = FLOAT                            ; rx power in mW
-        txbias                       = FLOAT                            ; tx bias in mA
-        laser_temperature	         = FLOAT                            ; laser temperature value in Celsius
-        prefec_ber                   = FLOAT                            ; prefec ber
-        postfec_ber                  = FLOAT                            ; postfec ber
-        cd_shortlink                 = FLOAT                            ; chromatic dispersion, high granularity, short link in ps/nm
-        cd_longlink                  = FLOAT                            ; chromatic dispersion, low granularity, long link in ps/nm
-        dgd                          = FLOAT                            ; differential group delay in ps
-        sopmd                        = FLOAT                            ; second order polarization mode dispersion in ps^2
-        pdl                          = FLOAT                            ; polarization dependent loss in db
-        osnr                         = FLOAT                            ; optical signal to noise ratio in db
-        esnr                         = FLOAT                            ; electrical signal to noise ratio in db
-        cfo                          = FLOAT                            ; carrier frequency offset in MHz
-        soproc                       = FLOAT                            ; state of polarization rate of change in krad/s
-        laser_config_freq            = FLOAT                            ; laser configured frequency in MHz
-        laser_curr_freq              = FLOAT                            ; laser current frequency in MHz
-        tx_config_power              = FLOAT                            ; configured tx output power in dbm
-        tx_curr_power                = FLOAT                            ; tx current output power in dbm
-        rx_tot_power                 = FLOAT                            ; rx total power in  dbm
-        rx_sig_power                 = FLOAT                            ; rx signal power in dbm
-        bias_xi                      = FLOAT                            ; modulator bias xi
-        bias_xq                      = FLOAT                            ; modulator bias xq
-        bias_xp                      = FLOAT                            ; modulator bias xp
-        bias_yi                      = FLOAT                            ; modulator bias yi
-        bias_yq                      = FLOAT                            ; modulator bias yq
-        bias_yp                      = FLOAT                            ; modulator bias yp
-        ========================================================================
+            Dictionary
         """
```

Comment:
@mihirpat1 should we rename this bulk api to dom status or as appropriate per the HLD table

Reply:
@prgeor I have addressed this now